tutorials/scripting/c_sharp/c_sharp_global_classes.rst

```diff
@@ -21,6 +26,35 @@ The ``MyNode`` type will be registered as a global class with the same name as t
 
 .. image:: img/globalclasses_addnode.webp
 
+The *Select a Node* window for the ``MyNode`` exported property filters the list
+of nodes in the scene to match the assignment restriction.
+
+.. code-block:: csharp
+
+    public partial class Main : Node
+    {
+        [Export]
+        public MyNode MyNode { get; set; }
+    }
+
+
+.. image:: img/globalclasses_exportednode.webp
+
+The inheritance hierarchy is used if a custom type isn't registered.
```

Comment:
This sentence is a bit unclear IMHO.

Reply:
Hmm, agreed. I'm thinking merging this with the `MySpecializedNode` paragraph is the right first step to making it clearer, I'll take a look.